pages/linux/reflector.md

```diff
@@ -0,0 +1,11 @@
+# reflector
+
+> Arch script to fetch and sort mirrorlists.
+
+- Get all mirrors, sort for download speed and save them:
+
+`sudo reflector --sort {{rate}} --save {{/etc/pacman.d/mirrorlist}}`
+
+- Only get the 10 recently sync'd German HTTPS mirrors sorted:
+
+`reflector --latest {{10}} --country {{Germany}} --protocol {{https}} --sort {{rate}}`
```

Comment:
Can we split the options into multiple examples and introduce them slowly ? Since we have space for a couple more examples, it becomes easy for the user to get ramped up slowly.